### ISSUE TITLE
Show the latest AC submission

### ISSUE THREAD

**Issue by 07-priyanshu** [07-priyanshu/Codeforces-Friends-Tracker#1]
Currently, the algorithm picks up the latest submission from the submission list of that person(CF ID) for a particular problem. The last submission in most cases is the AC one, if the problem was solved by that person but this is just an assumption.

In cases where the nth submission for the problem was AC and subsequent 2 submissions were made which were not AC but either WA,TLE, etc then the failed solution(n+2 th submission) is displayed. Instead,  the algorithm should filter out the latest AC solution(nth).

In short, modify the script to pick the latest **AC** submission instead of the latest submission. In case , no AC submission have been made, then the latest WA/TLE/etc submissions should be displayed.
